README.md

```diff
@@ -0,0 +1,52 @@
+# Q-Zandronum Flatpak edition
```

Comment:
no end user will see that readme.

Reply:
I put it just in case someone entered the github repository.